docs/cimpl6Authoring.md

```diff
@@ -39,56 +37,56 @@ Model key:
 
 * Define the use cases behind the creation of a model.
-* Create a high-level conceptual model which addresses your defined use case and can be easily understood by both technical and clinical communities.
-* Create a data dictionary listing the data elements, cardinality, and potential value sets involved if the data type is a coded element. This provides a convenient summary for implementers presented in a way that can be understood by non-technical subject matter experts involved in defining the use cases.
+* Create a high-level conceptual model to address your use case that can be easily understood by both technical and clinical communities.
+* Create a data dictionary listing the data elements, cardinality, and potential value sets involved if the data type is a coded element. This provides a convenient summary for implementers presented in a way that can be understood by non-technical subject matter experts.
 * Start documenting your model with CIMPL.
 
 ## Model Folder Structure and Source Control
 
-Create a folder/directory to contain all files you create for the model. If you are modeling with a distributed team, you should consider using a source control system for you CIMPL files.
+Create a folder/directory to contain all files you create for the model. If you are modeling with a distributed team, you should consider using a source control system for your CIMPL files.
 
-[CIMPL In-Depth Tutorial](#cimpl6Tutorial_detail.md)
+[CIMPL Tooling Reference Manual](cimpl6ToolingReference.md#suggested-directory-structure)
 
 ## Configuration File
 
-A configuration file sets parameters to drive IG creation.
+A Configuration file sets parameters to drive IG creation.
 
-By declaring optional parameters, your IG may include model documentation, a graphical view of the model, examples, or a data dictionary. These sections are automatically generated, are not required to include in the IG, but provide valuable information to IG consumers.
+By declaring optional parameters, your IG may include model documentation, a graphical view of the model, examples, and a data dictionary. These sections are automatically generated, are not required to include in the IG, but provide valuable information to IG consumers.
 
-The data dictionary export might require manual adjustments before the IG is published. Review this document to make sure it meets your needs.
+The data dictionary export might require manual adjustments before the IG is published. Review this document to make sure the data dictionary export meets your needs.
 
-[CIMPL Tooling Reference Guide](cimpl6ToolingReference.md)
+[CIMPL Tooling Reference Manual](cimpl6ToolingReference.md#configuration-file)
```

Comment:
Instead of the text saying "CIMPL Tooling Reference Manual", suggest changing to "Suggested Directory Structure for Your Project" or something along those lines.

Reply:
OK - made changes throughout.

Reply:
Those changes don't seem to have happened. I'm still seeing the same text of the links.